### ISSUE TITLE
Reorganisation of CBPP fields

### ISSUE THREAD

**Issue by drozas** [P2Pvalue/cbpp-directory-code#43]
Marco suggested to find a way to better capture the emergence of new categories (for those fields which were too rigid). David suggested the following strategy (as far as there is not going to be statistical analysis of those fields at the intermediate stage):

    Open those fields (tags-like), so the users can freely input them, and suggestions based on the current ones will be offered.
    During a second phase, the categories will be curated, and we can "set them as a rigid taxonomy" again. This will need to be manually curated by UAB (e.g. to avoid redundancies). At that point, it will be feasible to do statistical analysis again.
    UAB will take a decision about this, and then David will explore the implementation.

**Comment by drozas** [comment]
Waiting for final decision of UAB on this, and access to the document.

**Comment by drozas** [comment]
A) To reduce the barriers to add new cases:

A1) To reduce the psychological burden:
By default opening a window only with the compulsory fields; and putting at the end of the page in evidence the option "Add more information" that opens the other fields (that otherwise remain invisible).

A2) To simplify and reduce the time necessary:
Reducing the compulsory fields.
Actually the compulsory fields are:
Name *
URL *
General description *
Legal entity *
Language(s) of the platform *
Type of activity *
The most important type of collaboration *
Type of common resource resulting *
User generated content license *
Software platform license *

I suggest to eliminate as compulsory, leaving them as optional: Language(s) of the platform *, The most important type of collaboration * and also (but I am less sure) Legal entity *. I would instead just eliminate (see below why) User generated content license * and Software platform license *

In this case, the remaining compulsory fields would be only 5 (+1) and most very simple (the last two are with predefined options):
Name *
URL *
General description *
Type of activity *
Type of common resource resulting *
Plus eventually a sixth new field to be filled by text for describing the commons resource (see below).


B) To improve the quality or the type of information:

B1) In Type of activity * (that remains compulsory), at the option Others: when it is marked, it could be opened a space to specify (optionally) by text other possible areas of production not covered by our 28+1 categories.

B2) I would eliminate the last 4 fields:
User generated content license *
Holder of the content license
Software platform license *
Infrastructure provider

Because we have found these two categories (content and platform) in certain cases confusing and misleading.
I would substitute them with a space to describe by text the kind of commons produced, beyond its generic character as captured by the categories of Type of common resource resulting * (Resource, Service, Design, Methodology, Internet protocol, Brand, Others). Similarly to what we have done in our web collection.
We can make it compulsory or maybe better we can leave it as optional, but in any case I would open this field in the first page of the form to incentivize its filling (while the other optional information will not be visible without clicking on "Add more information").
I would also add a space – optional - to specify the license, using a window with the predefined options we used in the web collection.
